### ISSUE TITLE
Tests failing in master:  File `Zug-2010.epw` does not exist

### ISSUE THREAD

**Issue by daren-thomas** [architecture-building-systems/CityEnergyAnalyst#825]
Why is the cea even looking for that file?

Here is the whole output:

```
C:\Users\darthoma\Documents\GitHub\CityEnergyAnalyst (master)
λ cea test
failed to import from rc_model_sia_cc.pyd, falling back to pure python functions
failed to import from storagetank_cc.pyd, falling back to pure python functions
.  run_unit_tests
E........
======================================================================
ERROR: setUpClass (test_calc_thermal_loads.TestCalcThermalLoads)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\users\darthoma\documents\github\cityenergyanalyst\cea\tests\test_calc_thermal_loads.py", line 35, in setUpClass
    cls.weather_data = epwreader.epw_reader(weather_path)[
  File "c:\users\darthoma\documents\github\cityenergyanalyst\cea\utilities\epwreader.py", line 31, in epw_reader
    result = pd.read_csv(weather_path, skiprows=8, header=None, names=epw_labels).drop('datasource', axis=1)
  File "C:\Users\darthoma\AppData\Local\conda\conda\envs\cea\lib\site-packages\pandas\io\parsers.py", line 655, in parser_f
    return _read(filepath_or_buffer, kwds)
  File "C:\Users\darthoma\AppData\Local\conda\conda\envs\cea\lib\site-packages\pandas\io\parsers.py", line 405, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "C:\Users\darthoma\AppData\Local\conda\conda\envs\cea\lib\site-packages\pandas\io\parsers.py", line 764, in __init__
    self._make_engine(self.engine)
  File "C:\Users\darthoma\AppData\Local\conda\conda\envs\cea\lib\site-packages\pandas\io\parsers.py", line 985, in _make_engine
    self._engine = CParserWrapper(self.f, **self.options)
  File "C:\Users\darthoma\AppData\Local\conda\conda\envs\cea\lib\site-packages\pandas\io\parsers.py", line 1605, in __init__
    self._reader = parsers.TextReader(src, **kwds)
  File "pandas/_libs/parsers.pyx", line 394, in pandas._libs.parsers.TextReader.__cinit__ (pandas\_libs\parsers.c:4209)
  File "pandas/_libs/parsers.pyx", line 710, in pandas._libs.parsers.TextReader._setup_parser_source (pandas\_libs\parsers.c:8873)
IOError: File c:\users\darthoma\appdata\local\temp\reference-case-open\baseline\inputs\weather\Zug-2010.epw does not exist

----------------------------------------------------------------------
Ran 8 tests in 2.657s

FAILED (errors=1)
########################################
TaskFailed - taskid:run_unit_tests
Python Task failed: '<function run_unit_tests at 0x00000000046B5908>' returned False

(6192.0, 3216.0)
```

**Comment by daren-thomas** [comment]
OK. Seems someone changed the `locator.get_default_weather()` method to return a different value. I'm changing it back.

**Comment by daren-thomas** [comment]
Also, the polybox version changed and direct linking to the files does not seem to work anymore. I have added them to the repository. This makes everything easier anyway...

**Comment by daren-thomas** [comment]
Fixed! See PR #826
